starknet_py/tests/e2e/conftest.py

```diff
@@ -32,6 +33,32 @@
     "0x60D7C88541F969520E46D39EC7C9053451CFEDBC2EEB847B684981A22CD452E"
 )
 
+CONTRACT = """
```

Comment:
I'm not sure this should be here. Maybe another .cairo file in the mocked_contracts_dir?